### ISSUE TITLE
hello

### ISSUE THREAD

**Issue by bsfoolery** [soomeone/PsychsonMaker#1]
its work? maybe u post a video.

**Comment by stephenstockman** [comment]
Looks interesting but as bsfoolery stated it would be nice to see either a video or some other form of media (perhaps through more wiki documentation).

**Comment by soomeone** [comment]
Sure, I hadn't enough time to make all the stuff like wiki, documentation, video, etc. but I'm working on it ;)  Video may come the next days

But there is a problem: Recently I flashed on my USB drive (the one with the right controller) a firmware that was incompatible, so now I have a half bricked USB stick, where I can't really flash a new FW. I have to think out how to show it in the video...

**Comment by soomeone** [comment]
I'm sorry, I currently have much things to do. But it's coming soon....

**Comment by Rosalie241** [comment]
maybe add a seldf extracter in the TEMP folder? it is like 2 min of extra work I can do it too

**Comment by Rosalie241** [comment]
and it is really sad that  Psychson stopped with the project he could create a firmware patch that indeed just ported automaticly hmm sad but that is life :-/

**Comment by soomeone** [comment]
Yeah this is sad... But at least we have the current working release. Maybe sometime someone will continue the development.

Do you mean an extractor that extract the ZIP file (the "files" folder) in a temp dir, so you just have to start the .exe?

**Comment by Rosalie241** [comment]
Yes

**Comment by Rosalie241** [comment]
Is this project still alive?

**Comment by soomeone** [comment]
Yeah it is, but I didn't worked on it for some weeks. In addition my usb stick doesn't work right and I don't know where the problem is.
But yeah, I'm trying to keep my projects alive :)

**Comment by Rosalie241** [comment]
ok
